src/Mechanisms/DataStore.php

```diff
@@ -42,7 +42,7 @@ function has($instance, $key, $iKey = null) {
         }
 
         if ($iKey !== null) {
-            return !! $this->lookup[$instance][$key][$iKey] ?? false;
+            return !! ($this->lookup[$instance][$key][$iKey] ?? false);
```

Comment:
Unrelated, but it would be nice to use explicit casting here. So instead of `!!` using `(bool)` to return a boolean

Reply:
I agree, but I don't want to go changing code style unless I'm asked to

Reply:
Understandable. As this type of casting isn't used elsewhere in Livewire PHP code I thought it uncontroversial enough to bring up :)

Reply:
Let's see what Caleb wants :)